package-lock.json

```diff
@@ -6392,9 +6392,9 @@
       }
     },
     "eslint": {
-      "version": "6.3.0",
```

Comment:
This still needs to be reverted before landing. You can get the last version of package-lock before these changes by doing this:

```bash
git log package-lock.json
```

Then take that revision, and put it in this command:

```bash
git checkout <revision hash> -- package-lock.json
```

Commit the result and the changes should be reverted.

Reply:
Thanks @wlach, it appears that the changes have been reverted.